src/beautifiers/rustfmt.coffee

```diff
@@ -4,34 +4,50 @@ Requires https://github.com/nrc/rustfmt
 
 "use strict"
 Beautifier = require('./beautifier')
+path = require('path')
 
-module.exports = class Rustfmt extends Beautifier
+versionCheckState = false
 
+module.exports = class Rustfmt extends Beautifier
   name: "rustfmt"
 
   options: {
     Rust: true
   }
 
   beautify: (text, language, options) ->
-
-    # get file path which is the search path for rustfmt.toml as
-    # the beautifier runs rustfmt in a tmp directory.
-    # This will pick up any rustfmt.toml defined in the crate root
-
     editor = atom.workspace.getActivePaneItem()
-    file = editor?.buffer.file
-    filePath = file?.path
+    buffer = editor.getBuffer?()
+    filePath = buffer.getPath?()
```

Comment:
`filePath` is now accessible with https://github.com/Glavin001/atom-beautify/pull/956

Example usage:

```coffeescript
beautify: (text, language, options, context) ->
  filePath = context.filePath
```

See https://github.com/Glavin001/atom-beautify/blob/66be231b953a6efd496a1c4f56a33fc730c0b502/src/beautifiers/marko-beautifier.coffee#L26 and https://github.com/Glavin001/atom-beautify/blob/66be231b953a6efd496a1c4f56a33fc730c0b502/docs/add-languages-and-beautifiers.md#how-to-add-a-beautifier-for-a-language for more details.